examples/stocks/lib/stock_data.dart

```diff
@@ -56,9 +56,9 @@ class StockData extends ChangeNotifier {
 
   bool get loading => _httpClient != null;
 
-  void add(List<List<String>> data) {
-    for (List<String> fields in data) {
-      final Stock stock = new Stock.fromFields(fields);
+  void add(List<dynamic> data) {
+    for (List<dynamic> fields in data) {
+      final Stock stock = new Stock.fromFields(fields.cast<String>());
```

Comment:
I'd be tempted to leave this code as is and do the coercion at the call site on line 89.

Reply:
afaik there isn't a way to do the cast there that passes the current static analysis config as well as the runtime checks